setup.py

```diff
@@ -77,6 +76,7 @@
           'ExtensionClass >= 4.2.1',
           'Persistence >= 3.0a3',
           'RestrictedPython >= 5.1',
+          'RestrictedPython == 7.2a1.dev0; python_version > "3.12"',
```

Comment:
Having explicit versions in setup.py cannot be overridden anywhere else. So avoiding it was common practice. What's the reason behind needing this explicit version number? (For previous future Python releases it was not necessary, so we might have broken something in our chain.)

Reply:
The reasoning was an attempt to make it find and use that version. It did not work. I removed the pin. The PR will stay a draft and will not be merged until there is a RestrictedPython release.

Reply:
👍